src/lib/core/CHIPError.h

```diff
@@ -1697,7 +1696,61 @@ typedef CHIP_CONFIG_ERROR_TYPE CHIP_ERROR;
  *    The registered service name has collision on the LAN.
  *
  */
-#define CHIP_ERROR_MDNS_COLLISSION _CHIP_ERROR(180)
+#define CHIP_ERROR_MDNS_COLLISSION                            _CHIP_ERROR(180)
```

Comment:
is this re-arrangement of whitespace caused by clang-format, or other?

Reply:
just revert it back, it is my manual re-arrangment.

> Could we:
> 
> * split header into individual pieces (I stopped counting at 20 classes ... the header is huge)
Yep, will do this split in this ticket,
https://github.com/project-chip/connectedhomeip/issues/3644
> * add unit tests
Yep, will do this split in this ticket
https://github.com/project-chip/connectedhomeip/issues/3645,
> How will this interact with ZAP? is zap expected to generate encode/decode that use these classes?